src/auth/Login.js

```diff
@@ -0,0 +1,103 @@
+import React, { useContext, useState } from "react";
+import { Link, useNavigate } from "react-router-dom";
+//import { Link } from "react-router-dom";
+import { AuthContext } from "./AuthProvider";
+import { Box, Button, styled, TextField } from "@mui/material";
+
+const SignInButton = styled(Button)({
+  background: "#6fc4f9",
+  fontSize: "1.0rem",
+  border: 0,
+  borderRadius: 3,
+  color: "white",
+  padding: "10px 40px",
+  marginTop: "30px",
+  "&:hover": {
+    backgroundColor: "#57baf8",
+  },
+});
+
+const Login = ({ history }) => {
+  const { login } = useContext(AuthContext);
+  //AuthContextからlogin関数を受け取る
+  const navigate = useNavigate();
+
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    login(email, password, navigate);
```

Comment:
こちらは非同期処理で、呼び出しもawaitを使用しないと処理を待たずにプロセスが終了してしまうかと思いますので、修正すると良いです！
```suggestion
    await login(email, password, navigate);
```